components/resourceDetails/index.jsx

```diff
@@ -175,7 +172,7 @@ export default function ResourceDetails({
 
       {accessControl && ( // only show when we know user has control access
         <>
-          {isWacControlled && (
+          {isWac && (
```

Comment:
This would change to a function which takes the `accessControlType` and returns the boolean value you want here.